TestingLowerBounds/KullbackLeibler.lean

```diff
@@ -154,4 +164,115 @@ lemma kl_nonneg (μ ν : Measure α) [IsProbabilityMeasure μ] [IsProbabilityMea
 
 end kl_nonneg
 
+section Conditional
+
+variable {β : Type*} {mβ : MeasurableSpace β} {κ η : kernel α β} {μ : Measure α}
+
+open Classical in
+
+/--
+Kullback-Leibler divergence between two kernels κ and η conditional to a measure μ.
+It is defined as KL(κ, η | μ) := ∫ x, KL(κ x, η x) dμ.
+-/
+noncomputable
+def condKL (κ η : kernel α β) (μ : Measure α) : EReal :=
```

Comment:
Please update the blueprint with a link to this (and same comment with other significant lemmas about condKL).